tests/Api/FilesApiTest.php

```diff
@@ -0,0 +1,109 @@
+<?php
```

Comment:
Test should be run by Github Actions CI 

copy files from https://github.com/EscolaLMS/GH-Actions-Template and it should work out of the box, 

see other packages as reference, eg 
* https://github.com/EscolaLMS/H5P/actions
* https://github.com/EscolaLMS/H5P/tree/main/.github/workflows

Reply:
is it ok now?